utils/email.js

```diff
@@ -1,58 +1,29 @@
 import dotenv from 'dotenv-safe';
 import nodemailer from 'nodemailer';
-import { getPledgeInviteById } from '../airtable/request';
+import Constants from '../Constants';
 
 dotenv.config(); // Set up environment variables
 
-const SENDER_NAME = ''; // specify the name of the person sending an email, for example: "Nick Wong ⚡️🔋"
-
 const transporter = nodemailer.createTransport({
   host: 'smtp.gmail.com',
```

Comment:
this one's for my understanding: what does this mean?

Reply:
So basically, the `nodemailer` package uses an existing mail server to send emails. Gmail accounts can be used as that "existing mail server" so `nodemailer.createTransport` is setting up nodemailer with the gmail servers